cogs/tags/tags.py

```diff
@@ -577,7 +578,12 @@ async def generic(self, ctx: Context, name: str, *, content: str):
     async def generic_error(self, ctx: Context, error):
         if isinstance(error, commands.MissingRequiredArgument):
             await ctx.send("Tag " + str(error))
+        elif isinstance(error, commands.CheckFailure):
+            pass
+        elif isinstance(error, commands.UnexpectedQuoteError):
+            await ctx.send("Please do not use quotes in the tag name!")
         else:
+            await ctx.send("Something went wrong, please check the logs for details.")
```

Comment:
Just a nit-pick, but most users and admins can't see the logs. 
Wouldn't a "Something went wrong, please try again or contact the bot owner(s) if this continues." be a more fitting message?

Reply:
Sure, that's a good idea. I've updated it. Right now it will still actually print the default message this one (note the `raise error`). I still have to add logging in a later PR, after which I will remove the `raise` and keep this updated message.